web/impact/impact/graphql/types/expert_profile_type.py

```diff
@@ -132,8 +132,9 @@ def resolve_program_roles(self, info, **kwargs):
         without changing GraphQL queries on the front end.
         """
         user_roles_of_interest = [UserRole.FINALIST, UserRole.ALUM]
-        startup_roles_of_interest = [StartupRole.ENTRANT]
-        startup_roles_of_interest += StartupRole.WINNER_STARTUP_ROLES
+        startup_roles_of_interest = StartupRole.WINNER_STARTUP_ROLES
+        if info.context.user.is_staff:
```

Comment:
why not also check for `CLEARANCE_LEVEL_STAFF`

Reply:
`is_employee` will do this check.